src/bundle/Resources/public/js/tab_opener.js

```diff
@@ -24,5 +26,32 @@ $(function () {
             $('#tab-redirect-modal').modal('show');
             document.location = childTab.attr('href');
         }
+    } else {
+        // If reloading a page with a tab, reopen this tab:
```

Comment:
If a tab is open in a new window then change, this doesn't work.

Clicking on a tab should change the document location hash or remove it, or the priority could be given to the cookie, or…

Reply:
Done in b5c3bb84861ddea864ac8ca83bf0a8fbae650366 (without cookie, just URL hash)